mlflow/server/handlers.py

```diff
@@ -187,6 +188,38 @@ def _get_artifact_repo_mlflow_artifacts():
     return _artifact_repo
 
 
+def _get_trace_artifact_repo(trace_info: TraceInfo):
+    """
+    Resolve the artifact repository for fetching data for the given trace.
+
+    Args:
+        trace_info: The trace info object containing metadata about the trace.
+    """
+    artifact_uri = get_artifact_uri_for_trace(trace_info)
+
+    if _is_servable_proxied_run_artifact_root(artifact_uri):
+        # If the artifact location is a proxied run artifact root (e.g. mlflow-artifacts://...),
+        # we need to resolve it to the actual artifact location.
+        from mlflow.server import ARTIFACTS_DESTINATION_ENV_VAR
+
+        if not (path := _get_proxied_run_artifact_destination_path(artifact_uri)):
```

Comment:
```suggestion
        path = _get_proxied_run_artifact_destination_path(artifact_uri)
        if not path:
```

I'd use the normal assignment here, but this is just my preference.